lib/perl/Genome/Model/Tools/Annotate/AddRsid.pm

```diff
@@ -79,39 +92,70 @@ sub execute {
             my @var_alleles = @tmp1;
             @var_alleles = @tmp2 if (scalar(@tmp2) > scalar(@tmp1));
 
-            # Gets the list of allele frequencies if CAF nomenclature is used
-            if($INFO =~ /(CAF=[0-9.,]+)/) {
-                $caf_string = ($INFO =~ /(CAF=[0-9.,]+)/)[0];
-                @af = split(/,/, $caf_string);
-                # The first allele frequency provided is the reference allele which we don't want so drop it
-                shift @af;
+            # Script not designed to handle indels, ignores them if present
+            my $is_indel = 0;
+            for (@var_alleles) {
+                $is_indel = 1 if length($_) > 1;
+            }
+            if (length($ref) > 1 || $is_indel == 1) {
```

Comment:
The `== 1` here isn't strictly necessary.

One possible alternative is to eliminate `$is_indel` by labelling the while loop and changing the for loop like:
```perl
LINE: while (my $line = $vcf_fh->get_line) {
...
  for (@var_alleles) {
    next LINE if length($_) > 1;
  }
```